WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -26,6 +27,8 @@
 
 import javax.inject.Inject;
 
+import de.greenrobot.event.EventBus;
```

Comment:
Yes, we actually replaced LocalBroadcastReceiver by the event bus a long time ago, but we should do the switch and use event bus 3.0 everywhere.